easy_access/pdf/parse.py

```
@@ -95,17 +103,18 @@ async def process_pdf(pdf: PDF):
             )
 
         updatefields.extend(parsed_metadata.keys())
-        updatefields.append("extracted_text")
+        updatefields.append("extracted_text_id")
         updatefields.append("num_pages")
         return pdf, updatefields
 
     for pdf in pdfs:
         pdf, updatefields = await process_pdf(pdf)
```

Ruff (0.14.6)

Unpacked variable `updatefields` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)